README.md

```diff
@@ -61,8 +61,12 @@ Create an issue (eg "Add Ian's link") and assign it to yourself. Clone this repo
 - [Thierry Chau](https://quiet-bird-3083.fly.dev/)
 - [Kat Jarboe](https://melo616.github.io/)
 - [Jason Ho](https://jasonho404.github.io/links/)
+
+- [RJ      ] (https://www.mcdonalds.com/us/en-us.html)
```

Comment:
Remove the extra spaces in your name and between the end bracket ] and open parenthesis (, ie:
```md
[RJ](https://www.mcdonalds.com/us/en-us.html)
```

Reply:
done